pkg/trace/config/env.go

```diff
@@ -80,6 +80,14 @@ func loadEnv() {
 			log.Errorf("Bad format for %s it should be of the form '[{\"name\": \"tag_name\",\"pattern\":\"pattern\",\"repl\":\"replace_str\"}]', error: %v", "DD_APM_REPLACE_TAGS", err)
 		}
 	}
+	if v := os.Getenv("DD_APM_ADDITIONAL_ENDPOINTS"); v != "" {
+		ap := make(map[string][]string)
+		if err := json.Unmarshal([]byte(v), &ap); err != nil {
+			log.Errorf(`Could not parse DD_APM_ADDITIONAL_ENDPOINTS: %v. It must be of the form '{"https://my2.endpoint.eu": ["apikey1", ...], ...}'.`, err)
```

Comment:
nit: if this message is sent to our users, can we put a real endpoint instead of `my2.endpoint.eu`? like the real Datadog endpoint or something that is representative for a proxy.

Reply:
This shouldn't normally be seen by regular users because it's an undocumented configuration option, but sure, I've made the change. PTAL